doc/source/dev/development_environment.rst

```diff
@@ -228,10 +238,14 @@ And then in the debugger::
     (gdb) run
 
 The execution will now stop at the corresponding C function and you can step
-through it as usual.  With the Python extensions for gdb installed (often the
-default on Linux), a number of useful Python-specific commands are available.
+through it as usual. A number of useful Python-specific commands are available.
 For example to see where in the Python code you are, use ``py-list``.  For more
-details, see `DebuggingWithGdb`_.
+details, see `DebuggingWithGdb`_. Here lists the most commonly used commands::
+
+    list- List specified function or line
```

Comment:
I haven't checked how this is rendered, but perhaps a bulleted-list would be more appropriate than a literal block? It might be fine.

Reply:
@rossbar do you have a bulleted-list example?